packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -235,8 +236,9 @@ export class SlackBot {
                         appProfilePhotoUrl,
                     });
                 }
-
-                Sentry.captureException(e);
+                if (!(e instanceof ScreenshotError)) {
```

Comment:
https://lightdash.sentry.io/issues/6367556611/events/f2ce11226d53408eb2fb70ee3612096b/
this was duplicating ScreenshotErrors.
This ScreenshotError is thrown in `unfurlImage`>`saveScreenshot`